src/eva/tests/config/testIodaObsSpace.yaml

```diff
@@ -0,0 +1,106 @@
+diagnostics:
+- data:
+
+    # Data read
+    # ---------
+    type: IodaObsSpace
+    datasets:
+
+      - name: reference
+        filenames:
+          - aircraft.hofx.2020-12-14T21:00:00Z.nc4
+          - aircraft.hofx.2020-12-14T21:00:00Z.nc4
+        groups:
+          - name: GsiHofXBc
+            variables: [air_temperature, eastward_wind]
+          - name: GsiEffectiveQC
+            variables: [air_temperature, eastward_wind]
+
+      - name: experiment
+        filenames:
+          - aircraft.hofx.2020-12-14T21:00:00Z.nc4
+          - aircraft.hofx.2020-12-14T21:00:00Z.nc4
+        groups:
+          - name: hofx
+            variables: [air_temperature, eastward_wind]
+          - name: ObsValue
+            variables: [air_temperature, eastward_wind]
+          - name: EffectiveQC
+            variables: [air_temperature, eastward_wind]
+          - name: MetaData
+            variables: [air_pressure, dateTime, height, latitude, longitude, record_number,
+                        station_id]
+
+#      - name: reference_amsua
+#        filenames:
+#          - amsua_n19.hofx.2020-12-14T21:00:00Z.nc4
+#          - amsua_n19.hofx.2020-12-14T21:00:00Z.nc4
+#        channels: 2,3,14
+#        groups:
+#          - name: hofx
+#            variables: [brightness_temperature]
+#          - name: ObsValue
+#            variables: [brightness_temperature]
+#          - name: MetaData
+#            variables: [dateTime, height_above_mean_sea_level, latitude, longitude, record_number,
+#                        scan_position, sensor_azimuth_angle, sensor_view_angle, sensor_zenith_angle,
+#                        solar_azimuth_angle, solar_zenith_angle]
+
+
+
+  # Transforms
+  transforms:
+
+    - transform: arithmetic
+      arithmetic type: subtraction
+      collections: [reference]
+      variables: [air_temperature-*, eastward_wind-*]
```

Comment:
Is the wildcard here intended to mean 'all' variables (as in shell). Is there a reason why it wouldn't be fine to just do `air_temperature` since the `arithmetic variables` line handles this later?

Reply:
That might work actually. I think I was trying to imagine situations where the data class has put variables in with certain naming convention. I struggled a bit with how things should be named and it might be something that gets refined as the actual implementation of these transform classes takes shape.

Reply:
Yeah I think whatever you have that is working is fine for now and we can refine once we have real cases.